src/simplay-jupyter/src/utils.ts

```diff
@@ -14,3 +14,70 @@ export function createButton(
   classes.forEach(className => button.classList.add(className));
   return button;
 }
+
+export function tsToTime(ts: number): string {
+  let minutes = Math.trunc(ts / 60);
+  const seconds = ts - 60 * minutes;
+  const roundedSeconds =
+    seconds > 59.5 ? Math.trunc(seconds) : Math.round(seconds);
+  if (minutes >= 60) {
+    const hours = Math.trunc(minutes / 60);
+    minutes = minutes - 60 * hours;
+    return enhanceFullTimeStringWithZeros(hours, minutes, roundedSeconds);
+  }
+  return enhanceMinutesSecondsTimeStringWithZeros(minutes, roundedSeconds);
+}
+
+function enhanceMinutesSecondsTimeStringWithZeros(
```

Comment:
A more usual name would be padWithZeros (from to pad, in German "polstern") I think, but this is nitpicking.